core/chains/evm/forwarders/forwarder_manager_test.go

```diff
@@ -47,31 +42,30 @@ func TestFwdMgr_MaybeForwardTransaction(t *testing.T) {
 	linkAddr := common.HexToAddress("0x01BE23585060835E02B77ef475b0Cc51aA1e0709")
 	operatorAddr, _, _, err := operator_wrapper.DeployOperator(owner, ec, linkAddr, owner.From)
 	require.NoError(t, err)
-
-	forwarderAddr, _, _, err := authorized_forwarder.DeployAuthorizedForwarder(owner, ec, linkAddr, owner.From, operatorAddr, []byte{})
+	forwarderAddr, _, forwarder, err := authorized_forwarder.DeployAuthorizedForwarder(owner, ec, linkAddr, owner.From, operatorAddr, []byte{})
+	require.NoError(t, err)
+	ec.Commit()
+	_, err = forwarder.SetAuthorizedSenders(owner, []common.Address{owner.From})
 	require.NoError(t, err)
-
 	ec.Commit()
+	authorized, err := forwarder.GetAuthorizedSenders(nil)
+	require.NoError(t, err)
+	t.Log(authorized)
 
+	evmClient := client.NewSimulatedBackendClient(t, ec, testutils.FixtureChainID)
 	lp := logpoller.NewLogPoller(logpoller.NewORM(testutils.FixtureChainID, db, lggr, pgtest.NewPGCfg(true)),
-		cltest.NewSimulatedBackendClient(t, ec, testutils.FixtureChainID), lggr, 100*time.Millisecond, 2, 3)
-	fwdMgr := forwarders.NewFwdMgr(db, ethClient, lp, lggr, pgtest.NewPGCfg(true))
-	fwdMgr.ORM = forwarders.NewORM(db, logger.TestLogger(t), cfg)
+		evmClient, lggr, 100*time.Millisecond, 2, 3)
+	fwdMgr := NewFwdMgr(db, evmClient, lp, lggr, pgtest.NewPGCfg(true))
+	fwdMgr.ORM = NewORM(db, logger.TestLogger(t), cfg)
 
 	_, err = fwdMgr.ORM.CreateForwarder(forwarderAddr, utils.Big(*testutils.FixtureChainID))
 	require.NoError(t, err)
-
 	lst, err := fwdMgr.ORM.FindForwardersByChain(utils.Big(*testutils.FixtureChainID))
 	require.NoError(t, err)
 	require.Equal(t, len(lst), 1)
 	require.Equal(t, lst[0].Address, forwarderAddr)
 
-	// Mocking getAuthorisedSenders on forwarder to return EOA
```

Comment:
@essamhassan I think the mocking wasn't really necessary here? Simpler to just configure the contract since you already deployed it